flashinfer/gemm/gemm_base.py

```
@@ -2105,15 +2091,45 @@ def is_cudnn_override_shape_available() -> bool:
         return False
     try:
-        if cudnn.backend_version() < 92100:
+        backend_version = cudnn.backend_version()
+        if backend_version < 92100:
             return False
         version_str = cudnn.__version__
         major, minor = map(int, version_str.split(".")[:2])
-        return (major, minor) >= (1, 20)
+        required_frontend_version = (1, 24) if backend_version >= 92300 else (1, 20)
+        return (major, minor) >= required_frontend_version
     except Exception:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)